node/src/components/consensus/config.rs

```diff
@@ -11,3 +18,32 @@ pub struct Config {
     /// Path to secret key file.
     pub secret_key_path: External<SecretKey>,
 }
+
+/// Consensus protocol configuration.
+#[derive(DataSize, Debug)]
+pub(crate) struct ProtocolConfig {
+    pub(crate) highway_config: HighwayConfig,
+    /// Number of eras before an auction actually defines the set of validators.
+    /// If you bond with a sufficient bid in era N, you will be a validator in era N +
+    /// auction_delay + 1
+    pub(crate) auction_delay: u64,
+    pub(crate) unbonding_delay: EraId,
+    /// Name of the Genesis network.
```

Comment:
Not sure I understand.
```suggestion
    /// Name of the network.
```

Reply:
Done in [`3703796` (#665)](https://github.com/CasperLabs/casper-node/pull/665/commits/3703796ffe51d59e32e7d93af6573e4a1a4a0eea).